muranotest/Dockerfile

```diff
@@ -13,9 +16,10 @@ RUN python setup.py install
 RUN pip install nose tempest-lib
 WORKDIR /opt/murano
 RUN pip install -r test-requirements.txt
+COPY requirements.txt /opt/murano/requirements2.txt
```

Comment:
Where is this line? And I do not understand why replicating the requitements.txt file resolve this error